projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,8 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="10" class="tabs-viewport">
```

Comment:
`itemSize` seems to be rather 80

![grafik](https://user-images.githubusercontent.com/4947061/225277213-9a065951-b265-4567-8c38-a16dfa8f31e2.png)